services/registration/src/routes/statistics.ts

```diff
@@ -106,6 +106,54 @@ statisticsRouter.route("/").get(
               },
             },
           ],
+          // marketing data for applied applications
+          marketingData: [
+            {
+              $match: {
+                "status": { $ne: StatusType.DRAFT },
+                "applicationData.marketing": { $ne: null },
+              },
+            },
+            {
+              $group: {
+                _id: "$applicationData.marketing",
+                count: { $sum: 1 },
+              },
+            },
+          ],
+          // shirt size data for applied applications
+          shirtSizeData: [
+            {
+              $match: {
+                "status": { $ne: StatusType.DRAFT },
```

Comment:
for things like shirt size and dietary restrictions, this information is more important for confirmed applicants so that in future years, we can look back on this data and have a good idea of how much shirts/food to buy. Could you update those to reflect that?